src/general/OptionsMenu.cs

```diff
@@ -452,6 +452,26 @@ public void ApplySettingsToControls(Settings settings)
         customUsername.Editable = settings.CustomUsernameEnabled;
     }
 
+    // GUI Function, but it must be places before private ones?
+    public void OnBackPressed()
```

Comment:
I think this approach should not be taken. The main menu should instead detect if rebinding is in progress and skip doing anything on esc being pressed.

Reply:
Actually I think when rebinding is in progress BackButton should not be clicked as well... Fine though.

Reply:
I *think* you can't press the back button, because clicking the mouse would result in the keybinding being set to "mouse 1"

Reply:
That's true now, but I think it's a little strange. I was wondering if we could cancel binding when the mouse clicks not at that waiting for input button. If so, then clicking the button should not allow it to exit. So I think it makes sense to do so.

Reply:
I don't think it is possible to know if the player clicked the mouse on a button or not. So we just have to capture the entire click for binding the mouse buttons to work.